test/browser-tests/router-get/router-get.js

```diff
@@ -0,0 +1,296 @@
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// This is a test and we want descriptions to be useful, if this
+// breaks the max-length, it's ok.
+
+/* eslint-disable max-len */
+/* eslint-env browser, mocha */
+/* global testHelper */
+
+'use strict';
+
+describe('Test router.get method', () => {
+  let performFetch = (iframe, fetchUrl, expectedString) => {
+    return iframe.contentWindow.fetch(fetchUrl)
+      .then(response => {
+        response.status.should.equal(200);
+        return response.text();
+      })
+      .then(responseText => {
+        responseText.should.equal(expectedString);
+      });
+  };
+
+  let performTest = (swUrl, fetchUrl, expectedString, done) => {
+    let iframe = null;
+    let testPromise = testHelper.activateSW(swUrl)
+    .then(newIframe => {
+      iframe = newIframe;
+
+      // Call the iframes fetch event so it goes through the service worker
+      return performFetch(newIframe, fetchUrl, expectedString);
+    });
+
+    if (done) {
+      testPromise = testPromise.then(() => done(), done);
+    }
+
+    testPromise = testPromise.then(() => iframe);
+
+    return testPromise;
+  };
+
+  const serviceWorkersFolder = '/test/browser-tests/router-get/serviceworkers';
+
+  it('should return response for absolute url', done => {
+    performTest(
+      serviceWorkersFolder + '/relative.js',
+      '/test/relative-url-test',
+      '/test/relative-url-test',
+      done
+    );
+  });
+
+  it('should return response for relative url', done => {
+    performTest(
+      serviceWorkersFolder + '/relative.js',
+      serviceWorkersFolder + '/test/relative-url-test-2',
+      'test/relative-url-test-2',
+      done
+    );
+  });
+
+  it('should return the variable from a pattern', done => {
+    performTest(
+      serviceWorkersFolder + '/variable-match.js',
+      '/test/match/echo-this/pattern',
+      'echo-this',
+      done
+    );
+  });
+
+ // TODO: Find out correct behaviour https://github.com/GoogleChrome/sw-toolbox/issues/86
+  it.skip('should throw an error for route with an origin defined in sw', done => {
+    performTest(
+      serviceWorkersFolder + '/full-url.js',
+      location.origin + '/test/absolute-url-test',
+      '/test/absolute-url-test',
+      done
+    );
+  });
+
+  // TODO: Find out correct behaviour https://github.com/GoogleChrome/sw-toolbox/issues/86
+  it.skip('should throw an error for route with an origin defined in sw', done => {
```

Comment:
I know that this is a skipped test, but this has the same description as the previous test. Could you change it to distinguish how this test is different?